sensu/server/_handler_pagerduty.sls

```diff
@@ -0,0 +1,42 @@
+
+gem_sensu_pagerduty:
```

Comment:
There is but you probably won't have access to do this and it's going to be deprecated soon in favor of Mirantis packages and new CI/CD infrastructure.